docs/source/operator_reference_load.rst

```diff
@@ -9,7 +9,7 @@ Loading operators.
 .. raw:: html
 
     <iframe
-      src="_static/dpf_operators.html#CPython"
+      src="_static/dpf_operators.html"
```

Comment:
Found here: https://github.com/search?q=repo%3Aansys%2Fpydpf-core%20cpython&type=code

This file might need to be manually updated to help publication on Dev Portal: https://github.com/ansys/pydpf-core/blob/gh-pages/version/stable/operator_reference_load.html